task/01-strings-tasks.js

```diff
@@ -221,7 +226,21 @@ function getRectangleString(width, height) {
  *
  */
 function encodeToRot13(str) {
-    throw new Error('Not implemented');
+  const inputBase = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+  const outBase =   'NOPQRSTUVWXYZABCDEFGHIJKLMnopqrstuvwxyzabcdefghijklm';
+  let str2 = '';
+    for(let i = 0; i<str.length; i++) {
```

Comment:
It is necessary to follow the generally accepted code style: 
indentation after "for", "if", as well as indentation around operators "i <n".